virttest/qemu_vm.py

```diff
@@ -658,6 +658,10 @@ def add_nic(devices, vlan, model=None, mac=None, device_id=None,
                     if "rombar" in devices.execute_qemu("-device %s,?"
                                                         % model):
                         dev.set_param("rombar", 0)
+
+                # romfile can be '', i.e. to disable rom loading
+                if romfile is not None:
```

Comment:
As the target option would be omitted when its value is `None` I think it's safe to remove this check, what do you think?

https://github.com/avocado-framework/avocado-vt/blob/c40074ba6fe4c089f1418e9b44bae413c12bdcdd/virttest/qemu_devices/qdevices.py#L140-L143

Reply:
Done